src/xunit.analyzers/AssertEqualStringValueShouldNotUseGenericType.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Xunit.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class AssertEqualStringValueShouldNotUseGenericType : AssertUsageAnalyzerBase
+    {
+        private static readonly HashSet<string> EqualMethods = new HashSet<string>(new[] { "Equal" });
```

Comment:
Well, the original issue didn't mention it, and there is no string-specific overload of StrictEqual as there is with Equal. Maybe @bradwilson knows?